docs/content/en/docs/installation/_index.en.md

```diff
@@ -238,6 +217,21 @@ Please visit [OpenShift]({{%ref "_index.en.md" %}}) for more detailed informatio
 ```bash 
 $ kubectl -n fission apply -f \
     https://github.com/fission/fission/releases/download/{{% release-version %}}/fission-core-{{% release-version %}}-openshift.yaml
+$ cat << EOF | kubectl create -n my-kafka-project -f -
+# Source: fission-core/templates/deployment.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fission-fetcher
+  namespace: fission-function
+---
+# Source: fission-core/templates/deployment.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fission-builder
+  namespace: fission-builder
+EOF
```

Comment:
I am sorry I don't understand, the fission-builder SA already exists on fission/fission@1.11.2 fission-all-1.11.2.yaml (download), why are we creating this again

Reply:
When we specify kubectl -n fission apply ... , all the resources in other namespaces aren't created and the service accounts are in different namespaces from fission, kubectl informs about this.